makefiles/tools/jlink.inc.mk

```diff
@@ -12,6 +12,24 @@ RESET_FLAGS ?= reset
 
 JLINK_SERIAL ?= $(DEBUG_ADAPTER_ID)
 
+JLINK_IF ?= SWD
```

Comment:
why this default value? wasn't the case before

Reply:
This is the default in `jlink.sh`:
https://github.com/RIOT-OS/RIOT/blob/741b9d3b2da71b3b259498ca9d4f062da48b2e82/dist/tools/jlink/jlink.sh#L70

Reply:
Missed that thanks.

Reply:
although why set it again then?

Reply:
I found it was simpler this way but I'll change that.

Reply:
addressed